packages/shared/src/components/onboarding/OnboardingFeedHeader.tsx

```diff
@@ -62,13 +68,25 @@ export const OnboardingFeedHeader = ({
       event_name: AnalyticsEvent.CreateFeed,
     });
 
-    return router.replace({
-      pathname: router.route,
-      query: {
-        welcome: 'true',
-      },
-    });
-  }, [registerLocalFilters, router, trackEvent, updateAlerts]);
+    if (!onboardingOptimizations) {
+      router.replace(
+        isExtension
+          ? 'https://app.daily.dev/?welcome=true'
```

Comment:
```suggestion
          ? `${webappUrl}/?welcome=true`
```

Reply:
Done. I knew there was something like that, but I didn't find it when I looked 😬😓